graphql_compiler/schema_transformation/utils.py

```diff
@@ -342,14 +344,15 @@ def check_ast_schema_is_valid(ast):
     """
     try:
         schema = build_ast_schema(ast)
-    except Exception as e:  # Can't be more specific -- see graphql/utils/build_ast_schema.py
+    except Exception as e:  # Can't be more specific -- see graphql/utilities/build_ast_schema.py
```

Comment:
My impression is that this is resolved in the new version of the GraphQL library, so we can remove the `try-except` entirely and just have a bare `schema = build_ast_schema(ast)`. No need to rewrap the exception now that it isn't of type `Exception`.

Reply:
It's probably fine either way, my gut feeling is to go for simplicity and just let the underlying error bubble up. The alternative would be to do something like `raise SchemaStructureError("Input is not a valid schema.") from e` and that might be okay too — though in that case we should probably tighten down what kinds of exceptions we expect. But that seems like a lot of work for not that much benefit at all.